client/src/Components/Flows/TippingFlow/EnterReceiverCode/EnterReceiverCode.js

```diff
@@ -99,8 +107,10 @@ const EnterCode = (props) => {
 
 const mapStateToProps = (state) => {
   return {
-    receiverSearchResultsArray: state.TipReceiverReducer.receiverSearchResultsArray,
-    token: state.UserReducer.token
+    ReceiverSearchResultsArray: state.TipReceiverReducer.receiverSearchResultsArray,
+    UserReducer: state.UserReducer,
```

Comment:
Not sure about naming your props as "xReducer". Is there a specific reason for this? 🤔

Reply:
No particular reason. I find that it makes for a clear overview of the data, especially since there's a `user` object nested inside `userReducer`. I've thought about combining the two, but it made for a bit of a messy experience. 

If need be, I can adjust it.